packages/xo-web/src/xo-app/sr/tab-general.js

```diff
@@ -183,7 +184,7 @@ export default class TabGeneral extends Component {
         )
       }
       // search root base copy for each VDI
-      const vdisInfo = vdis.map(({ id, parent, name_label, usage }) => {
+      const vdisInfo = vdis.map(({ id, parent, name_label, usage, $VBDs }) => {
```

Comment:
`$VBDs` is not used.

Reply:
We use it  to get `vbds` in `UsageTooltip`.

Reply:
You're right :+1: